Dockerfile.stable

```diff
@@ -1,82 +1,82 @@
 FROM cfmeqe/sel_base_fc29
 
-#TODO: figure why user 0 was needed
-USER 0
+ENV SELENIUM_VERSION=3.141.59 \
+    SELENIUM_HOME=/home/selenium \
+    SELENIUM_PORT=4445 \
+    REVERSE_PROXY_PORT=4444 \
+    FIREFOX_VERSION=60.4.0esr \
```

Comment:
please update version to the latest one 60.7.0 and appropriate geckodriver version